docs/source/get_started/credentials.rst

```diff
@@ -1,9 +1,21 @@
+Accessing Credentials and API
+=============================
+
 Accessing Credentials
-=====================
+---------------------
 Below are instructions for accessing Superstaq credentials. Note that you will need a Google account to access Superstaq.
 
-#. Navigate to https://superstaq.super.tech/.
+#. Navigate to https://superstaq.infleqtion.com/.
 #. Click "Sign in with Google".
```

Comment:
Should we highlight 'Login via Superstaq' also? Though I think Google is preferred here

Reply:
Good point - I'll update that. Thanks!!